src/test/regress/expected/isolation_tenant_isolation.out

```diff
@@ -135,15 +133,14 @@ step s2-print-cluster:
 nodeport|shardid|success|result
 ---------------------------------------------------------------------
    57637|1500009|t      |     0
-   57637|1500010|t      |     1
+   57637|1500010|t      |     0
```

Comment:
why do these appear?

Reply:
We are isolating tenant with id=5 inside a transaction. Then, before committing it, we also try to delete id=5 from another transaction which does not see the effects of first transaction. It fetches old shards and waits for the locks. Then first transaction commits, and second transaction acquires locks and checks if shards are still valid. It reroutes shards because they changed and successfully deletes id=5. 

Before we were failing with a shard not found error, but now we execute delete successfully. That is why we have those changes. 

```sql
step s2-begin:
 BEGIN;

step s2-isolate-tenant:
 SELECT isolate_tenant_to_new_shard('isolation_table', 5, shard_transfer_mode => 'block_writes');

isolate_tenant_to_new_shard
---------------------------------------------------------------------
                    1500010
(1 row)

step s1-delete:
 DELETE FROM isolation_table WHERE id = 5;
 <waiting ...>
step s2-commit: 
 COMMIT;

step s1-delete: <... completed>
```

Reply:
got it, we may want to cover a non-fast-path query

Reply:
added